com.gluonhq.scenebuilder.metainfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.gluonhq.scenebuilder</id>
+  <name>Gluon Scene Builder</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>0BSD</project_license>
+  <developer_name>Gluon HQ</developer_name>
+  <summary>Drag and Drop, Rapid Application Development.</summary>
+  <description><p>Gluon Scene Builder is a drag and drop UI designer tool allowing rapid desktop and mobile app development. Scene Builder separates design from logic, allowing team members to quickly and easily focus on their specific aspect of application development.
```

Comment:
This should be split into multiple `<p>` tags. Additionally, please change the disclaimer to follow the one used by e.g. https://beta.flathub.org/apps/details/com.jetbrains.WebStorm